source/ncengine/graphics2/diligent/resource/GlobalResourceSignature.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "Common/interface/RefCntAutoPtr.hpp"
+#include "Graphics/GraphicsEngine/interface/RenderDevice.h"
+#include "Graphics/GraphicsEngine/interface/DeviceContext.h"
+#include "Graphics/GraphicsEngine/interface/PipelineResourceSignature.h"
+
+#include <memory>
+
+namespace nc::graphics
+{
+class GlobalTextureBufferResource;
+
+class GlobalResourceSignature
+{
+    public:
+        static constexpr auto BindingIndex = 0u;
+        static constexpr auto SignatureName = "GlobalTextureBuffer";
```

Comment:
This signature will hold all of our global resources, not just Textures. I'd rename to something like "Global Resource Signature"

Reply:
If this isn't just a cleanup/old code thing and you're confused by my comment, see this discussion:
https://github.com/DiligentGraphics/DiligentEngine/issues/161
I'm imagining something like ^ this for this class.

Disregard if you are too

Reply:
Oops, copy paste error